finishFeature/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
You can use this plugin to generate appropriate and full gitignore files :) CMake needed if you have Native code (C++)
![image](https://user-images.githubusercontent.com/46504259/146927290-63757af6-f17e-42f5-8794-1d6c7d85a9ac.png)
![image](https://user-images.githubusercontent.com/46504259/146927351-4839cfc1-04cb-444a-92a8-e2c12c66eb54.png)
![image](https://user-images.githubusercontent.com/46504259/146927473-a06d218e-c6ec-47b8-8189-d19f7e3edd55.png)

Reply:
Oh, thank you very much. Kolya recommend me to use some website service to add git ignore files :)